packages/docs/community/release-process/base-cli-contractkit-dappkit-utils.md

```diff
@@ -0,0 +1,81 @@
+# Release Process for Base, Celocli, ContractKit, DAppKit, and Utils
+
+## Versioning
+
+Use the standard MAJOR.MINOR.PATCH semantic versioning scheme described at [semver.org](http://semver.org).
+
+New releases can be expected as follows:
+  * Major releases: approximately yearly 
+  * Minor releases: approximately 8 times a year
+  * Patch releases: as needed
+
+Development builds will be identified as such: `x.y.z-dev`, and will be published as `x.y.z` when stable. 
+
+## Identifying releases
+
+### NPM
+
+You can find the npm packages in the following places:
+  * [@celo/base](https://www.npmjs.com/package/@celo/base)
+  * [@celo/celocli](https://www.npmjs.com/package/@celo/celocli)
+  * [@celo/contractkit](https://www.npmjs.com/package/@celo/contractkit)
+  * [@celo/dappkit](https://www.npmjs.com/package/@celo/dappkit)
+  * [@celo/utils](https://www.npmjs.com/package/@celo/utils)
+
+### Github tags
+
+To identify the commits included in a specific release and see which new features were added or bugs fixed, please refer to the [release notes](https://github.com/celo-org/celo-monorepo/releases) in the monorepo. Also to keep track of continual updates to the stable and dev versions of the packages, each package has a `CHANGELOG.md` file: [Base](https://github.com/celo-org/celo-monorepo/blob/master/packages/base/CHANGELOG.md), [Celocli](https://github.com/celo-org/celo-monorepo/blob/master/packages/cli/CHANGELOG.md), [Contractkit](https://github.com/celo-org/celo-monorepo/blob/master/packages/contractkit/CHANGELOG.md), [Dappkit](https://github.com/celo-org/celo-monorepo/blob/master/packages/dappkit/CHANGELOG.md), [Utils](https://github.com/celo-org/celo-monorepo/blob/master/packages/utils/CHANGELOG.md).
+All releases should be tagged with the version number, e.g. `contractkit-vX.Y.Z`. Each release should include a summary of the release contents, including links to pull requests and issues with detailed description of any notable changes. 
+
+### Communication
+
+The community will be notified of package updates through the following channels: 
+
+For all releases: 
+  * Each package’s `CHANGELOG.md` file, as mentioned above
+  * Github releases page, as mentioned above
+  * [Discord](https://discord.gg/6yWMkgM): #developer-chat, #mainnet, and #sdk
+
+For major releases: 
+  * Twitter: [@CeloDevs](https://twitter.com/CeloDevs)
+  * Mailing list: cLabs’ Tech Sync
+  * [Celo Forum](https://forum.celo.org/)
+
+
+
+## Build process
+
+The packages are published by following the instructions [here](https://github.com/celo-org/celo-monorepo/blob/master/README-dev.md).
+
+## Testing
+All builds of these packages are automatically tested for performance and backwards compatibility in CI. Any regressions in these tests should be considered a blocker for a release.
```

Comment:
Not a question about this PR and non-blocking, but wondering generally (and sorry if this has been extensively discussed elsewhere!): do we already have any tests that verify version consistency stuff? Maybe that’s something we could incorporate either into normal tests or into a publish script in the future, very much out of scope for this PR. i.e. checks internal consistency of versioning and breaks if it’s off, ensures a basic change log entry is present, version matches what we expect of major/minor/patch, that kind of thing? @barbaraliau

Reply:
I don't believe we have that. @medhak1 do you know? if we don't, that would be a nice script to add to the backlog.

Reply:
We have a script that makes sure all dependencies are pointing to a real version, i think `yarn check:packages`, but we can definitely improve upon this and add stuff around changelogs, etc!